ipatests/azure/Dockerfiles/Dockerfile.build.fedora

```diff
@@ -4,10 +4,6 @@ ENV container=docker LANG=en_US.utf8 LANGUAGE=en_US.utf8 LC_ALL=en_US.utf8
 
 ADD dist /root
 RUN echo 'deltarpm = false' >> /etc/dnf/dnf.conf \
-    && dnf install -y dnf-plugins-core \
-    && dnf config-manager --set-enabled updates-testing \
-    && dnf config-manager --set-enabled updates-testing-modular \
-    && dnf config-manager --set-disabled fedora-cisco-openh264 \
```

Comment:
Why is this change necessary? Is this because the images differ?

Reply:
this seems to be unnecessary with the fedora:32 image anymore.

Reply:
That was added by me in 26f96595b010396f055c7d11a645be57fc637863 :
> Azure: Make dnf repos consistent
    Build container(image registry.fedoraproject.org/f32/fedora-toolbox)
    has two more dnf repos enabled compared to Tests container(image
    fedora:32). This results in the packages built within the Build
    container can have dependencies which are unresolvable(missing)
    within Tests container.

With this change `build` stage and `tests` one will use the same parent image.

Reply:
ack for this, thanks for the explanation.